docs/docs/sample-scripts/tenant/monitor-notify-healthstatus/index.mdx

```diff
@@ -20,7 +20,7 @@ Following is the overview of the script package
 
 2. If there is an outage or some of the service is not normal, we will be adding the information to SharePoint list using the command [spo listitem add](https://pnp.github.io/cli-microsoft365/cmd/spo/listitem/listitem-add/)
    1. Advantage of adding to SharePoint list - You can configure Power Automate for List item Added so that you can define your business process if needed
-3. Script also will send an email to the configured user/s using the command [spo mail send](https://pnp.github.io/cli-microsoft365/cmd/spo/mail/mail-send/) so that concerned team could be notified
+3. Script also will send an email to the configured user/s using the command [outlool mail send](https://pnp.github.io/cli-microsoft365/cmd/outlook/mail/mail-send/) so that concerned team could be notified
```

Comment:
```suggestion
3. Script also will send an email to the configured user/s using the command [outlook mail send](https://pnp.github.io/cli-microsoft365/cmd/outlook/mail/mail-send/) so that concerned team could be notified
```